### PR TITLE
Typo in "session" documentation.mdx

### PR DESCRIPTION
**Pull request by Lucas-Athayde**

### CODE REVIEW

File: docs/api/commands/session.mdx [hunk 1]
```diff
@@ -322,7 +322,7 @@ cy.session('user', () => {
 ### Caching session data across specs
 
 If you want to use the same session across multiple specs in the same Cypress
-run on the same machine, add `cacheAcrossSpecs=true` to the session options to
+run on the same machine, add `cacheAcrossSpecs:true` to the session options to
```
Comment (AtofStryker) [inline docs/api/commands/session.mdx:325]: @Lucas-Athayde I am not entirely sure this is a typo, rather trying to display that the value should be set to true. Do we know if this is consistent with Cypress' current documentation style?

**Review by AtofStryker — CHANGES_REQUESTED** [review]
@Lucas-Athayde thank you for opening a PR! Could you please fill out the PR description on the proposed change and why you are making it?

File: docs/api/commands/session.mdx (hunk 1 above)
Comment (Lucas-Athayde) [reply docs/api/commands/session.mdx:325]: Hey there @AtofStryker, thank you for your comment and reviewing my PR.
You've raised a good point, after checking out other parts of the documentation I could see that you're indeed correct!
It's actually displaying the value that is has to be set to.
Sorry for any inconvenience.

**Comment by Lucas-Athayde** [comment]
Not really a typo in documentation, but a way to point out the correct value.
Closing PR as there's no change to be made.

**Pull request closed without merging**